posthog/hogql/database/database.py

```diff
@@ -881,10 +930,14 @@ def serialize_database(
     # Process views using prefetched data
     views_dict = {view.name: view for view in all_views}
     for view_name in views:
-        view: SavedQuery | None = getattr(context.database, view_name, None)
+        view: Table | TableGroup | None = getattr(context.database, view_name, None)
         if view is None:
             continue
 
+        # Don't need to process TableGroups, they're already processed below
+        if isinstance(view, TableGroup):
+            continue
```

Comment:
This is not needed after the fix in `database.py:290` but it's an extra safety measure